### ISSUE TITLE
wav2vec 2.0 pre-training get stuck when using a 10k hours dataset, works fine with only 1k hours

### ISSUE THREAD

**Issue by YamSok** [pytorch/fairseq#3718]
#### What is your question?

I am trying to pre-train wav2vec 2 on a custom dataset (10k hours). When pre-training on the complete dataset with the "_base_librispeech_" config, **the loss value get stuck at 6.658 and accuracy is oscillating from 0.2 up to 0.8** (exact same loss value as in #3661). The same problem occurs when pre-training with the same config but using only 1k hours from the dataset.

Seeing #3661, I tried to lower the lr from 0.0005 to 0.0001, which works but only when using 1k hours. The initial behavior still occurs when trying this solution for a training on the complete dataset. 

Is this behavior to be expected when training with this much data and no change to the '_base_librispeech_' config? And if so, how should I adjust hyperparameters to account for that ?

Custom daset info : 

- French speakers audio
- segmented using a CNN to supress silences and music
- 16 kHz, 16 bits, mono .wav files
- Avg. clip duration : 30 sec.
- Cumul. duration : 10k hours

#### Code

Config 1 : Running fairseq-hydra-train with default parameters and no change to the '_base_librispeech_' config.
Config 2 : Running fairseq-hydra-train with default parameters and lowering lr from 0.0005 to 0.0001 in the '_base_librispeech_' config. 

#### What have you tried?

Lowering lr from 0.0005 to 0.0001 in the '_base_librispeech_' config

#### What's your environment?

 - fairseq Version (e.g., 1.0 or master): master
 - PyTorch Version (e.g., 1.0): 1.9.0
 - OS (e.g., Linux): Redhat Entreprise Linux 8.1
 - How you installed fairseq (`pip`, source): source
 - Build command you used (if compiling from source):
 - Python version: pip install --editable ./
 - CUDA/cuDNN version: 3.7.10
 - GPU models and configuration: 8x Nvidia Tesla V100 
 - Any other relevant information:

**Comment by jubick1337** [comment]
Hi @YamSok 
I'm facing similar issues about pretraining w2v2
Have you tested running with ```optimization.update_freq='[8]'```?
In my case it helped

**Comment by RuABraun** [comment]
Also having similar problems, gonna try out increasing value of update_freq thanks @jubick1337

**Comment by bmilde** [comment]
What exactly is optimization.update_freq='[8]' doing? @jubick1337

**Comment by bmilde** [comment]
Ok got it, its for simulating multi-GPU:

> Note: you can simulate 64 GPUs by using k GPUs and adding command line parameters (before --config-dir) distributed_training.distributed_world_size=k +optimization.update_freq='[x]' where x = 64/k
